algoliasearch-common/src/main/java/com/algolia/search/APIClient.java

```diff
@@ -342,7 +342,7 @@ public CreateUpdateKey addApiKey(@Nonnull ApiKey key, @Nonnull RequestOptions re
             .setData(key));
   }
 
-  /** Deprecated: use updateApiKey */
+  /** Deprecated: use {@link #updateApiKey(String, ApiKey)} */
```

Comment:
Javadoc seems cool. Is it part of Javadoc to also include the `Deprecated` note (as it's already an annotation below)?

Reply:
No it's only me.